### DISCUSSION TITLE
Text editor

### DISCUSSION THREAD

**Discussion by uforek** [Ulauncher/Ulauncher#1206] [Q&A] [answered]
When opening a file from the launcher, it defaults to gedit. Is there a way to configure this behavior in the launcher or do I need to modify my OS to default to a different editor?

**Comment by friday** [comment, marked as answer]
If you're using systemd, https://github.com/Ulauncher/Ulauncher/discussions/1351

If not just set the EDITOR variable or your desktop default apps. For example right click -> "open with..." -> "always open with..."

**Comment by uforek** [comment]
> If you're using systemd, see #991 "Setting environment variables for Ulauncher when running the ulauncher service via systemd"
> 
> If not just set the EDITOR variable or your desktop default apps. For example right click -> "open with..." -> "always open with..."

Much appreciated!
